wnfs-common/README.md

```diff
@@ -0,0 +1,42 @@
+<div align="center">
+  <a href="https://github.com/wnfs-wg" target="_blank">
+    <img src="https://raw.githubusercontent.com/wnfs-wg/rs-wnfs/main/assets/logo.png" alt="WNFS Logo" width="100" height="100"></img>
+  </a>
+
+  <h1 align="center">wnfs-common</h1>
+
+  <p>
+    <a href="https://crates.io/crates/wnfs-common">
+      <img src="https://img.shields.io/crates/v/wnfs-common?label=crates" alt="Docs">
+    </a>
+    <a href="https://codecov.io/gh/wnfs-wg/rs-wnfs">
+      <img src="https://codecov.io/gh/wnfs-wg/rs-wnfs/branch/main/graph/badge.svg?token=95YHXFMFF4" alt="Code Coverage"/>
+    </a>
+    <a href="https://github.com/wnfs-wg/rs-wnfs/actions?query=">
+      <img src="https://github.com/wnfs-wg/rs-wnfs/actions/workflows/checks.yaml/badge.svg" alt="Build Status">
+    </a>
+    <a href="https://github.com/wnfs-wg/rs-wnfs/blob/main/LICENSE">
+      <img src="https://img.shields.io/badge/License-Apache%202.0-blue.svg" alt="License">
+    </a>
+    <a href="https://docs.rs/wnfs">
+      <img src="https://img.shields.io/static/v1?label=Docs&message=docs.rs&color=blue" alt="Docs">
+    </a>
+    <a href="https://discord.gg/zAQBDEq">
+      <img src="https://img.shields.io/static/v1?label=Discord&message=join%20us!&color=mediumslateblue" alt="Discord">
+    </a>
+  </p>
+</div>
+
+<div align="center"><sub>:warning: Work in progress :warning:</sub></div>
+
+##
+
+<!-- Todo: Summary here -->
+
+## Usage
+
+<!-- Todo: Usage description here -->
+
+```rust
+todo!()
+```
```

Comment:
Intended leftover TODOs, or should they be attended to this PR?

Reply:
imo we can add that on another  PR

Reply:
Also I think I would recommend having examples directory and treating them as artifacts
docs.rs has learned to harves them and inject into relevant places
e.g. we could remove it maybe